recipes/boost/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "1.60.0" %}
+{% set filename = "boost_%s.tar.bz2" % version.replace(".", "_") %}
+
+package:
+  name: boost
+  version: {{ version }}
+
+source:
+  fn:  {{ filename }}
+  url: http://sourceforge.net/projects/boost/files/boost/{{ version }}/{{ filename }}
```

Comment:
The filename could be defined as a jinja variable *after* the ``source:`` section? That way it is in one place, readable, and doesn't pollute the sacred first few lines... otherwise, I :heart: it